compiler/rustc_mir_transform/src/coverage/graph.rs

```diff
@@ -214,6 +247,36 @@ impl CoverageGraph {
             None
         }
     }
+
+    /// For each loop that contains the given node, yields the "loop header"
+    /// node representing that loop, from innermost to outermost. If the given
+    /// node is itself a loop header, it is yielded first.
+    pub(crate) fn loop_headers_containing(
+        &self,
+        bcb: BasicCoverageBlock,
+    ) -> impl Iterator<Item = BasicCoverageBlock> + Captures<'_> {
```

Comment:
just a side note here, the [Rust 1.82 release notes mention](https://blog.rust-lang.org/2024/10/17/Rust-1.82.0.html#precise-capturing-use-syntax) that we now have dedicated `use<>` syntax to replace `Captures`, and this eventually becoming the default in Rust 2024.
Though I believe `+ '_` would also work here, as the `&self` lifelime is the only one captured here?

I also don’t really know what the current conventions for compiler code are related to all this.

Reply:
Oh, is that stable now? Nice.

I think I'm going to stick with `Captures` for now. After the compiler switches over to Rust 2024, someone will probably go through and try to rip out unnecessary uses of `Captures`, and it will be easier to find and migrate this code if it matches the prevailing style in the rest of the compiler.